### PR TITLE
[WIP] ecommerce Basket Page -> micro-frontend

### PR DESCRIPTION
**Pull request by UvgenGen**
https://github.com/openedx/public-engineering/issues/68
[OeX_Depr-158](https://youtrack.raccoongang.com/agiles/104-960/105-2571?issue=OeX_Depr-158) ecommerce Basket Page -> micro-frontend

### CODE REVIEW

File: ecommerce/core/models.py [hunk 1]
```diff
@@ -376,6 +370,7 @@ def enterprise_grant_data_sharing_url(self):
 
     @property
     def payment_domain_name(self):
+        # TODO: update it
```
Comment (idegtiarov) [inline ecommerce/core/models.py]: It seems it is a time for updating, since self argument is removed

File: ecommerce/tests/factories.py [hunk 2]
```diff
@@ -41,6 +41,7 @@ class Meta:
     enable_embargo_check = False
     enable_partial_program = False
     discovery_api_url = 'http://{}.fake/'.format(Faker().domain_name())
+    payment_microfrontend_url = 'http://{}.fake/'.format(Faker().domain_name())
```
Comment (idegtiarov) [inline ecommerce/tests/factories.py]: nit: I would use f-string instead of format?

**Review by idegtiarov — CHANGES_REQUESTED** [review]
Nits and comment, other LGTM! Please take a look at my comment inline

**Review by idegtiarov — APPROVED** [review]
LGTM thanks!

File: ecommerce/core/management/commands/create_or_update_site.py [hunk 3]
```diff
@@ -144,13 +144,6 @@ def add_arguments(self, parser):
...
                             required=True,
                             help='URL for Discovery service API calls.')
-        parser.add_argument('--enable-microfrontend-for-basket-page',
-                            action='store',
-                            dest='enable_microfrontend_for_basket_page',
-                            type=bool,
-                            required=False,
-                            help='Use the microfrontend implementation of the '
-                                 'basket page instead of the server-side template')
         parser.add_argument('--payment-microfrontend-url',
```
Comment (dyudyunov) [inline ecommerce/core/management/commands/create_or_update_site.py:147]: I think this option should be `required=True` now, doesn't it?

Comment (UvgenGen) [reply ecommerce/core/management/commands/create_or_update_site.py:147]: updated)

**Review by dyudyunov — APPROVED** [review]

**Review by idegtiarov — APPROVED** [review]
LGTM

**Review by ruzniaievdm — APPROVED** [review]

**Pull request closed without merging**